localstack-core/localstack/openapi.yaml

```diff
@@ -514,8 +588,6 @@ paths:
       - $ref: '#/components/parameters/SnsPhoneNumber'
       responses:
         '204':
-          content:
-            text/plain: {}
```

Comment:
Q: is this because the body is empty? If so, is there another way to state the response mimetype?